uptane-standard.md

```diff
@@ -780,15 +790,19 @@ The ECU SHALL verify that the latest image matches the latest metadata as follow
 
 If the ECU has secondary storage, the checks SHOULD be performed on the image in secondary storage, before it is installed.
 
-If any step fails, the ECU SHALL jump to the fifth step ({{create_version_report}}). If a step fails and the ECU does not have secondary storage, and the ECU has created a backup of its previous working image, the ECU SHOULD now install the backup image.
+If any step fails, the ECU SHALL jump to the fifth step ({{create_version_report}}). If a step fails and the ECU does not have secondary storage, then the ECU SHALL ensure a backup of a previous working image is stored before attempting the verification. Upon failure, the ECU SHALL install the backup image to ensure it is in a working condition.
+
+#### Install image {#install_image}
+
+The ECU SHALL attempt to install the update at a time when all pre-conditions are met. These pre-conditions MAY include ensuring the vehicle is in a safe environment for install (e.g., vehicle is in Park when updating a specific ECU). Another pre-condition MAY include ensuring the ECU has a backup of its current image and metadata in case the current installation fails.
```

Comment:
```suggestion
The ECU SHOULD attempt to install the update at a time when all pre-conditions are met. These pre-conditions MAY include ensuring the vehicle is in a safe environment for install (e.g., vehicle is in the parked state when updating a specific ECU). Another pre-condition MAY include ensuring the ECU has a backup of its current image and metadata in case the current installation fails.
```

Reply:
SHALL should be reserved for security relevant to Uptane, not necessarily the vehicle safety, which is outside its scope

Reply:
To avoid making the entire section conditional, I rewrote it to fit your recommendation.
I think we can have another discussion on the relationship between vehicle safety & cybersecurity, but we can save that until the 14th :)